File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

    std::string convert(int number) {
        std::string line = "";

        if (number % 3 == 0) {
            line = line + "Pling";
        }
        if (number % 5 == 0) {
            line = line + "Plang";
        }
        if (number % 7 == 0) {
            line = line + "Plong";
        }

        if (line == "") {
            return std::to_string(number);
        } else {
            return line;
        }
    }

}

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
	if number % 3 != 0 && number % 5 != 0 && number % 7 != 0 {
		return fmt.Sprint(number)
	}

	value := ""

	if number % 3 == 0 {
		value += "Pling"
	}
	if number % 5 == 0 {
		value += "Plang"
	}
	if number % 7 == 0 {
		value += "Plong"
	}

	return value
}
